torch_xla/experimental/spmd_fully_sharded_data_parallel.py

```diff
@@ -192,3 +209,49 @@ def _auto_wrap(
             "if using an `auto_wrap_policy`")
 
     recursive_wrap(**auto_wrap_kwargs, **fsdp_kwargs)
+
+
+def _cast_floats_tensors(dtype: torch.dtype, *args: Any,
```

Comment:
can we directly import from xla_fully_sharded_data_parallel.py?

Reply:
fixed